frontend/src/metabase/account/login-history/components/LoginHistory/LoginHistory.jsx

```diff
@@ -18,7 +18,7 @@ import {
 } from "./LoginHistory.styled";
 
 const LoginHistoryItem = item => (
-  <Card py={1} px="20px" my={2}>
+  <Card py={1} px="20px" my={2} key={item.timestamp}>
```

Comment:
I think this might lead to errors. Since these cannot be modified, would it be better to use an index instead? Furthermore, I think it's incorrect to set `key` inside the component being rendered, and it should be set when the component is used e.g. `items.map((item, index) => <LoginHistoryItem key={index} item={item} />`